### ISSUE TITLE
Eyre does not honor Clay permissions

### ISSUE THREAD

**Issue by belisarius222** [urbit/arvo#657]
As noted by @ohAitch in a comment on https://github.com/urbit/arvo/issues/656, Eyre does not honor the Clay permissions. It should not serve files if the requesting user does not have permission. This is a security hole at the moment.

**Comment by Fang-** [comment]
Heh, I was just about to go make this.

Note that when this gets implemented, we'll want to set appropriate default permissions for `/web` and its contents on boot/eyre-startup, and/or have eyre print a warning when `|serve`ing a path for which permissions have been set to fully private.

**Comment by belisarius222** [comment]
Does fully private need a warning? Seems like a reasonable setting for diaries and other personal files.

**Comment by Fang-** [comment]
Right, fully private is iirc still accessible by the owner, so that's valid then. Never mind me there.
